src/data.js

```diff
@@ -1117,6 +1117,27 @@ module.exports = [
       'Golang',
     ],
   },
+  {
+    name: 'Rikin Patel',
+    description: 'Experienced Front-end developer, Passionate about Javascript',
+    url: 'https://patelrikin.com/uses',
```

Comment:
Why don't we make this https://patelrikin.com/#uses

Reply:
I wasnt sure if it would pass validation

Reply:
@HugoDF made changes